ivy/functional/frontends/tensorflow/linalg.py

```diff
@@ -184,6 +184,20 @@ def lu_matrix_inverse(lower_upper, perm, validate_args=False, name=None):
     )
 
 
+def lu_reconstruct(lower, upper):
+    L, U = ivy.array(lower), ivy.array(upper)
+
+    if L.shape[0] != L.shape[1] or U.shape[0] != U.shape[1] or L.shape[0] != U.shape[0]:
+        raise ValueError(
+            "Invalid dimensions for LU reconstruction. `l` and `u` must be square"
+            " matrices of the same size."
+        )
+
+    A = ivy.matmul(L, U)
+
+    return A
+
```

Comment:
Hi @jdboachie, thanks for the PR. You will also need to add a frontend test to be considered for merging.